typing/ctype.mli

```diff
@@ -127,6 +127,7 @@ val associate_fields:
         (string * field_kind * type_expr) list *
         (string * field_kind * type_expr) list
 val opened_object: type_expr -> bool
+val object_row: type_expr -> type_expr
```

Comment:
I missed the fact that this function was not exposed, sorry. There is already `Btype.row_of_type` which achieve the same result for `Tobject _`.